src/input/SimModSuite.h

```diff
@@ -357,6 +357,54 @@ struct get_second : public std::unary_function<MyMap::value_type, int>
     }
 };
 
+enum elementType {
+  vertex = 0,
+  edge = 1,
+  face = 2,
+  region = 3
+};
+
+void setMeshSize(pGModel model, pACase &meshCase, MeshAttributes &MeshAtt) {
+    // Set mesh size on vertices, edges, surfaces and regions
+    const char *element_name[4] = { "vertex", "edge", "face", "region" };
+
+    for (int elType = vertex; elType != region+1; elType++ )
```

Comment:
With a class enum this could be:
`for (const auto elType : {ElementType::Vertex, ElementType::Face, ...}) {`